scripts/query.js

```diff
@@ -0,0 +1,45 @@
+const dotenv = require("dotenv")
```

Comment:
the variables are already in the container passed from the docker-compose. why is it needed to use dotenv?